apis/events/v1alpha1/eventsource_types.go

```diff
@@ -52,9 +52,10 @@ type EventSourceSpec struct {
 	Build *openfunction.BuildImpl `json:"build,omitempty"`
 }
 
-// SinkSpec describes an event source for the Kafka.
+// SinkSpec describes an event source for the Kafka, priority Uri > Reference.
```

Comment:
@arugal Thanks for the PR, the test part is professional.

The original comment is wrong actually, would you help to correct it?
`// SinkSpec describes an event source for the Kafka, priority Uri > Reference.`
=>
`// SinkSpec specifies the receiver of the events an EventSource received, sinks in the Uri format have higher priority than sinks in Reference format.`

Reply:
Done